invenio/modules/access/upgrades/access_2015_05_06_accROLE_accACTION_accARGUMENT_id.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Upgrade recipe."""
+
+from invenio.legacy.dbquery import run_sql
+from invenio.modules.upgrader.api import op
+
+depends_on = ['invenio_release_1_2_0']
+
+
+def info():
+    """Info."""
+    return "Add new column id in accROLE_accACTION_accARGUMENT table."""
+
+
+def do_upgrade():
+    """Implement your upgrades here."""
+    # table accROLE_accACTION_accARGUMENT
```

Comment:
try http://docs.sqlalchemy.org/en/latest/core/reflection.html and `hasattr(table.c, 'id')`